vortex-serde/src/layouts/write/layouts.rs

```diff
@@ -36,6 +40,16 @@ impl Layout {
     pub fn column(children: Vec<Layout>) -> Self {
         Self {
             id: COLUMN_LAYOUT_ID,
+            buffers: None,
```

Comment:
i suppose this is where we could shove shared dicts in the future?

Reply:
I think that would be a DictLayout?